common/cont-lib/usr/share/cont-docs/70-general.txt.tpl

```diff
@@ -0,0 +1,16 @@
+General container help
+----------------------
+
+Run `docker run {% if privileged %} --privileged {% endif -%} {{ container }}{% if 'container-usage' not in cmd %} container-usage{% endif %}` to get this help.
+
+Run `docker run -ti {% if privileged %} --privileged {% endif -%} {{ container }} bash` to obtain interactive shell.
+
+Run `docker exec -ti {% if privileged %} --privileged {% endif -%} CONTAINERID bash` to access already running container.
```

Comment:
@hhorak IMO yes.  Because the general info about the container describes ATM
three cases which should - even on *-systemtap images - be OK to run without
--privileged.

Or we can say something like:

```
{% if privileged %}
This container is expected to be run like:
    'docker run --privileged {{ container }}'
{% endif %}
```